backend/.env.dist

```diff
@@ -21,4 +21,6 @@ JWT_SECRET=random
 WEB_FRONTEND_URL=http://localhost:3000
 KEY_GEN_BUFFER_SIZE=10
 KEY_GEN_HOSTNAME=kgs1-staging.short-d.com
-KEY_GEN_PORT=443
\ No newline at end of file
+KEY_GEN_PORT=443
+
+AUTH_TOKEN_DURATION=604800000000
```

Comment:
This number doesn't tell me anything. Is it seconds, minutes? It is better to add a brief comment or reflect it in the variable's name `AUTH_TOKEN_DURATION_IN_SEC`

Reply:
Thank you for the comment, I change it to AUTH_TOKEN_LIFETIME_IN_SEC!